style.css

```diff
@@ -0,0 +1,70 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap");
+
+.navbar {
+  display: flex;
+  justify-content: end;
+  list-style-type: none;
+}
+
+.page1 {
+  display: flex;
+  flex-direction: column;
+  background-image: url('/img/image.png');
+  background-size: cover;
+  background-repeat: repeat-x;
+  margin: 5%;
+}
+
+.navbar-tag {
+  padding-left: 30px;
+  padding-right: 30px;
+  text-decoration: none;
+  display: flex;
+  justify-content: space-between;
+  justify-items: baseline;
+}
+
+.logo {
+  padding: 15px 0 0 30px;
+}
+
+.header {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 60px;
+  color: #091e42;
```

Comment:
- [ ] On this milestone you are to build the mobile site not the desktop ,the mobile header consist of the logo and the hamburger menu button.
- [ ] 
![Screenshot 2023-06-02 140846](https://github.com/medijdir/portfolioM1_project/assets/92458236/320e3720-54b0-4964-b6f8-ca693d82fcef)